ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -129,6 +129,19 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-workers-rhcos-rhel8
+- as: e2e-aws-ipi-ovn-hypershift-p2
+  cron: 0 4 */7 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@aws-ipi and @network-ovnkubernetes and not @proxy and not @disconnected
```

Comment:
No need `not @proxy and not @disconnected`.
E.g, for a test case only works on disconnected cluster, it has disconnected tag.
if a test case only works on connected cluster, it has connected tag.
if the test cases works on both connected and disconnected cluster, then it has both connected tag and disconnected tag.
`not @disconnected` will exclude those test cases that can work on both connected cluster and disconnected cluster.